CONTRIBUTING.md

```diff
@@ -7,10 +7,23 @@ This doc is intended for contributors to `cadence-java-client` (hopefully that's
 ## Development Environment
 
 * Java 11 (currently, we use Java 11 to compile Java 8 code).
-* Thrift 0.9.3
-* Gradle build tool
+* Thrift 0.9.3 (use [homebrew](https://formulae.brew.sh/formula/thrift@0.9) or [distribution](https://downloads.apache.org/thrift/0.9.3/))
+* Gradle build tool [6.x](https://github.com/uber/cadence-java-client/blob/master/gradle/wrapper/gradle-wrapper.properties)
 * Docker
 
+:warning: Note 1: You must install the 0.9.x version of Thrift. Otherwise compiling would fail at error `error: package org.apache.thrift.annotation does not exist`
+
+:warning: Note 2: It's currently compatible with Java 8 compiler but no guarantee in the future. 
+
+## IntelliJ IDE integration (Optional)
+
+* Make sure you set the gradle path with the right version ([currently 6.x](https://github.com/uber/cadence-java-client/blob/master/gradle/wrapper/gradle-wrapper.properties))
+
+![Screen Shot 2021-10-01 at 4 57 45 PM](https://user-images.githubusercontent.com/4523955/135696878-81c1e62e-eb04-45e6-9bcb-785ac38b6607.png)
```

Comment:
would be better to change the names from screenshot to some valid description of the image